### ISSUE TITLE
Extra autoregressive sampling parameters currently not covered by API

### ISSUE THREAD

**Issue by thomashopf** [evedesignbio/evedesign#5]
* EVmutation2: min_p sampling and sample_gaps
* General: PSSM bias, include/exclude AAs

Probably doesn't make sense to handle globally as we would need to hack into the autoregressive sampler of each method (ideally would be able to modify logits flexibly independent of model before sampling next token e.g. with a callback)

**Comment by thomashopf** [comment]
these go to individual models, pssm bias is in system spec
